sdk/nanotdf_test.go

```diff
@@ -239,3 +240,89 @@ func NotTestCreateNanoTDF(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestGetECPublicKeyKid(t *testing.T) {
+	var tests = []struct {
+		name       string
+		kasURL     string
+		dialOption grpc.DialOption
+		shouldFail bool
+	}{
+		{
+			name:       "Valid URL, Unreachable gRPC server",
+			kasURL:     "http://localhost",
+			dialOption: grpc.WithBlock(),
+			shouldFail: true,
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			_, _, err := getECPublicKeyKid(test.kasURL, test.dialOption)
+			if (err != nil) != test.shouldFail {
```

Comment:
We do use [testify](https://github.com/stretchr/testify) in other tests but it is fine to not.

Reply:
ugh, testify just isn't my jam